internal/linker/link_node_modules.bzl

```diff
@@ -19,7 +19,7 @@ def _debug(vars, *args):
         print("[link_node_modules.bzl]", *args)
 
 # Arbitrary name; must be chosen to globally avoid conflicts with any other aspect
-_ASPECT_RESULT_NAME = "link_node_modules__aspect_result"
+ASPECT_RESULT_NAME = "link_node_modules__aspect_result"
```

Comment:
If we're depending on this outside of this file, lets change the name to `MODULE_MAPPINGS_ASPECT_RESULTS_NAME` perhaps to match `module_mappings_aspect`.

Also, lets ensure that the module mappings we're depending on will be forward compatible to 4.0 since I'll be removing the map to tuple (type, path) and make it just just a map to path

Reply:
Renamed to `MODULE_MAPPINGS_ASPECT_RESULTS_NAME`